docs/get-started.mdx

```diff
@@ -0,0 +1,231 @@
+builds Netdata from its source code.
+
+Copy the script, paste it into your node's terminal, and hit `Enter` to begin the installation process.
+
+<OneLineInstall />
+
+Next, [open and explore the dashboard](#open-and-explore-the-dashboard).
+
+## Run Netdata with Docker
+
+You can also try out Netdata's capabilities in a [Docker container](/packaging/docker/README.md). Copy the following
+`docker run` command into your node and hit `Enter` to download and run the container.
+
+```bash
+docker run -d --name=netdata \
+  -p 19999:19999 \
+  -v netdataconfig:/etc/netdata \
+  -v netdatalib:/var/lib/netdata \
+  -v netdatacache:/var/cache/netdata \
+  -v /etc/passwd:/host/etc/passwd:ro \
+  -v /etc/group:/host/etc/group:ro \
+  -v /proc:/host/proc:ro \
+  -v /sys:/host/sys:ro \
+  -v /etc/os-release:/host/etc/os-release:ro \
+  --restart unless-stopped \
+  --cap-add SYS_PTRACE \
+  --security-opt apparmor=unconfined \
+  netdata/netdata
+```
+
+Next, [open and explore the dashboard](#open-and-explore-the-dashboard).
+
+## Open and explore the dashboard
+
+To start using Netdata, open a browser and navigate to `http://NODE:19999`, replacing `NODE` with either `localhost` or
+the hostname/IP address of a remote node.
+
+![Animated GIF of navigating to the
+dashboard](https://user-images.githubusercontent.com/1153921/80825153-abaec600-8b94-11ea-8b17-1b770a2abaa9.gif)
+
+Netdata's dashboard is a single page with hundreds of preconfigured charts and thousands of metrics that update every
+second. Everything you see is autodetected and organized with zero configuration required.
+
+To see more charts, scroll down or use the **menu** on the right-hand side of the dashboard to navigate between
+**sections**, which are groups of closely related charts.
+
+![Animated GIF of using the menus and
+submenus](https://user-images.githubusercontent.com/1153921/80832425-7c528600-8ba1-11ea-8140-d0a17a62009b.gif)
+
+## Interact with charts
+
+Every chart in the Netdata dashboard is fully interactive, and Netdata synchronizes your interactions to help you
+understand exactly how a node behaved in any timeframe, whether that's seconds or days.
+
+![Animated GIF of the standard Netdata dashboard being manipulated and synchronizing
+charts](https://user-images.githubusercontent.com/1153921/80839230-b034a800-8baf-11ea-9cb2-99c1e10f0f85.gif)
+
+Let's cover two of the most important ways to interact with charts: panning through time and zooming.
+
+To pan through time, **click and hold** (or touch and hold) on any chart, then **drag your mouse** (or finger) to the
+left or right. Dragging to the right pans backward through time, while dragging to the left pans forward in time. Think
+of it like pushing the current timeframe off the screen so you can see what came before or after.
+
+To zoom, press and hold `Shift` or `Alt`, then use your mouse's scroll wheel, or a two-finger pinch if you're using a
```

Comment:
In MacOS it's only Shift. Should we give instructions per OS (Windows, linux, macos) ?